spec/javascript/channels/client_channel.spec.js

```diff
@@ -1,20 +1,20 @@
 import {callback, getChannelName} from "channels/client_channel";
```

Comment:
If we were to further refactor this to look pretty similar to a railsy spec, it might make sense to have a single "class-like" import and then a test for each method on the "class". In this scenario, adding `getChannelName` as a method to whatever the `callback` object is (maybe we should call it a `ClientChannel` object?) then we could say
```javascript
describe ("ClientChannel"), () => {
  describe(".received")
  // ...
  describe(".channelName")
}
```
```